spyder/app/mainwindow.py

```diff
@@ -2442,11 +2442,11 @@ def render_issue(self, description='', traceback=''):
 
 ## Package Versions
 
-* Spyder {spyder_version} `{commit}`

```

Comment:
Is there a reason you removed the backticks here around the commit?

Reply:
Because if you're not running Spyder from a git repo then `commit` is empty.

Reply:
Oh, my fault. At some point, those were only inserted if Spyder was a dev build, but sometime between the initial version, the rebase, and the final one the logic changed. Thanks for the catch.